cla_backend/settings/base.py

```diff
@@ -316,8 +317,12 @@ def env_var_truthy_intention(name):
 CALL_CENTRE_NOTIFY_EMAIL_ADDRESS = os.environ.get("CALL_CENTRE_NOTIFY_EMAIL_ADDRESS", DEFAULT_EMAIL_TO)
 
 NON_ROTA_HOURS = {"weekday": (datetime.time(8, 0), datetime.time(17, 0))}
+DISCRIMINATION_NON_ROTA_HOURS = {"weekday": (datetime.time(8, 0), datetime.time(18, 0))}
 
-NON_ROTA_OPENING_HOURS = OpeningHours(**NON_ROTA_HOURS)
+# Opening hours for a provider of a particular category, the key matches the category key
+NON_ROTA_OPENING_HOURS = defaultdict(
+    lambda: OpeningHours(**NON_ROTA_HOURS), {"discrimination": OpeningHours(**DISCRIMINATION_NON_ROTA_HOURS)}
```

Comment:
This is clever. But it took me a little while to parse. There's a couple of things going on here at once, and in addition this is a defaultdict, which we're not really dealing with all that often so have probably slightly forgotten how they work.

I think maybe, at the expense of conciseness, we could consider doing it in two steps:
1) initialise the defaultdict with the lamba for its default value
2) update the dict with the discrimination key:value pair